gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -237,7 +232,7 @@ private SeekableByteChannel newReadChannel(Path path, Set<? extends OpenOption>
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method newWriteChannel() has an NPath complexity of 336](https://www.codacy.com/app/mziccard/gcloud-java/file/2423054461/issues/source?bid=3123107&fileBranchId=3239177#l232)